node/src/gcp/value.rs

```diff
@@ -0,0 +1,383 @@
+use google_datastore1::api::{ArrayValue, Entity, Key, LatLng};
+use std::collections::HashMap;
+
+use super::error::ConvertError;
+
+#[derive(Debug, Clone)]
+pub enum Value {
+    BooleanValue(bool),
```

Comment:
Is it hard to create a place for common code between FastAuth and Multichain codebases? Code duplication is never a good idea. But I understand, we do not want to touch the old codebase.

Reply:
Yeah I don't want to mess up mpc code since it's working and I'm not sure maybe we retire mpc code base altogether someday so decided to write a new one.